packages/framework/src/validators/json-schema.validator.ts

```diff
@@ -60,10 +60,16 @@ export class JsonSchemaValidator implements Validator<JsonSchema> {
     } else {
       return {
         success: false,
-        errors: (validateFn.errors as ErrorObject<string, Record<string, unknown>, unknown>[]).map((err) => ({
-          path: err.instancePath,
-          message: err.message as string,
-        })),
+        errors: (validateFn.errors as ErrorObject<string, Record<string, unknown>, unknown>[]).map((err) => {
+          const nestedPath = err.instancePath.split('/').join('.');
+          const requiredFieldName = err.params?.missingProperty as string;
+
+          return {
+            path: err.instancePath,
+            message: err.message as string,
+            property: `${nestedPath}${requiredFieldName ? `.${requiredFieldName}` : ''}`,
```

Comment:
For missing required fields, we have to manually had the field name. It is not part of the instancePath. Otherwise we won't show those "required" missing errors on the input